_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5130,6 +5135,13 @@ mfxU64 get_background_color(const mfxVideoParam & videoParam)
 {
     for (mfxU32 i = 0; i < videoParam.NumExtParam; i++)
     {
+        if (videoParam.ExtParam[i]->BufferId == MFX_EXTBUFF_VPP_COLORFILL)
+        {
+            mfxExtVPPColorFill *extCF = reinterpret_cast<mfxExtVPPColorFill *>(videoParam.ExtParam[i]);
+            if (extCF->Enable != MFX_CODINGOPTION_ON) {
```

Comment:
there id function ```IsOn```

Reply:
We have 3 copies of IsOn() function. I added another one for vpp. I thing this function shuld be moved into api/include/mfxstructures.h file (where MFX_CODINGOPTION_* definitions are defined)